### ISSUE TITLE
Video directory share

### ISSUE THREAD

**Issue by FixTechToday** [DanielnetoDotCom/YouPHPTube#267]
I'm currently having 2 sites with YouPHPtube, one for gaming, one for normal videos. When someone uploads an video from the Gaming site (embedded from YT), it also appears on the main site. But the thumbnail doesn't, because that's in another directory. My question is; can I make 1 public directory where both sites get there video data from?

**Comment by DanielnetoDotCom** [comment]
I do not think it may work, you can share the videos dir, but you will have database problem. the encoder updates only 1 streamer database per video

**Comment by FixTechToday** [comment]
Clear! Second question; I think your also familiar with YT's dark mode. Is there maybe a user-side thema switcher, so people can choose their own wanted theme?

**Comment by DanielnetoDotCom** [comment]
It is on the road map to make a plugin for that

**Comment by FixTechToday** [comment]
Awesome! Maybe an feature for verified badges/accounts? (Like YT has)

**Comment by DanielnetoDotCom** [comment]
Hi, 
What is badges/accounts?

**Comment by FixTechToday** [comment]
Like this: 
<img width="337" alt="schermafbeelding 2017-12-12 om 20 58 07" src="https://user-images.githubusercontent.com/21104166/33905534-322f3e4a-df7f-11e7-8d0f-cf63579580ff.png">

The little verified tick near the name. That would be awesome to separate normal users from verified users like admins and bigger channels

**Comment by DanielnetoDotCom** [comment]
verified users means email verified?

**Comment by FixTechToday** [comment]
No, I mean like, a option to verify people. On YT, that means that the YouTuber is connected to a network so he/she can earn money. 👍

**Comment by ychoucha** [comment]
Hello van u five US a link of i website

To sée how that's works 

Thanks

**Comment by DanielnetoDotCom** [comment]
@FixTechToday I have put this requirement on our TODO List https://github.com/DanielnetoDotCom/YouPHPTube/issues/293
